lib/ansible/parsing/mod_args.py

```diff
@@ -287,7 +288,9 @@ def parse(self):
 
         # walk the input dictionary to see we recognize a module name
         for (item, value) in iteritems(self._task_ds):
-            if item in BUILTIN_TASKS or item in action_loader or item in module_loader:
+            if any([item in BUILTIN_TASKS,
```

Comment:
Better use logical `or` so that checks would be performed sequentially as needed. Current implementation performs check functions invocations first even if it's not needed actually.